lib/repo/repo.ex

```diff
@@ -179,13 +179,19 @@ defmodule ExAudit.Repo do
       end
 
       # ExAudit.Repo behaviour
+
+      @deprecated "Use history_query/1 instead"
       def history(struct, opts \\ []) do
```

Comment:
@narrowtux should we keep or deprecate this one?

Reply:
Keep. The history function will also render complete structs from the diffs, but it needs the entire history for that.